applications/StructuralMechanicsApplication/custom_utilities/structural_mechanics_element_utilities.cpp

```diff
@@ -0,0 +1,47 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Philipp Bucher
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "structural_mechanics_element_utilities.h"
+#include "includes/variables.h"
+
+namespace Kratos {
+namespace StructuralMechanicsElementUtilities {
+
+bool ComputeLumpedMassMatrix(
```

Comment:
i think we shall decide if it is in properties or in processinfo, 

i don't love the idea of looking in both places. What would happen if the decision is contradictory?